packages/documentation/src/utils/figma-link.ts

```diff
@@ -0,0 +1,6 @@
+const fileId = 'xZ0IW0MJO0vnFicmrHiKaY';
+const figmaMode: 'design' | 'dev' = 'dev';
+
+export function getComponentFigmaLink(figmaNodeId: string) {
+  return `https://www.figma.com/file/${fileId}/Components-Post?type=design&node-id=${figmaNodeId}&mode=${figmaMode}&t=j3o3HMwq5nigL5Mj-4`;
```

Comment:
Unfortunately, it seems there is no way to zoom-in into the frame (see for example https://forum.figma.com/t/link-to-frame-fit-the-frame-to-screen/35680, there are multiple requests from the community).